src/cpp/session/modules/SessionProfiler.R

```diff
@@ -90,7 +90,7 @@
 {
    tryCatch({
       resources <- .rs.profileResources()
-      htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/")
+      suppressWarnings(htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/"))
```

Comment:
Note that `normalizePath` is a no-op if no such path exists so if you _really_ want a normalized path then you're just hiding an error.